### PR TITLE
Support Django 2.n 'view' perm with Django 1.n backward compatibility

### PR DESCRIPTION
**Pull request by fzzylogic**

### CODE REVIEW

File: admin_tools/menu/items.py [hunk 1]
```diff
@@ -189,7 +189,7 @@ def init_with_context(self, context):
         items = self._visible_models(context['request'])
         apps = {}
         for model, perms in items:
-            if not (perms['change'] or perms['view']):
+            if not (perms['change'] or perms.get('view', False)):
```
Comment (izimobil) [inline admin_tools/menu/items.py]: I think this is wrong, IMO it should be:
`if not perms['change'] and not perms.get('view', False)`

Or maybe a better solution would be to do like in dashboard/modules.py:
`if perms['change'] or perms.get('view', False)`

File: admin_tools/menu/items.py [hunk 2]
```diff
@@ -288,7 +288,7 @@ def init_with_context(self, context):
         """
         items = self._visible_models(context['request'])
         for model, perms in items:
-            if not (perms['change'] or perms['view']):
+            if not (perms['change'] or perms.get('view', False)):
```
Comment (izimobil) [inline admin_tools/menu/items.py]: Same remark as above.

File: admin_tools/menu/items.py (hunk 1 above)
Comment (izimobil) [reply admin_tools/menu/items.py]: Nevermind, I misread the if, your condition is correct !

**Comment by izimobil** [comment]
@fzzylogic , @johanneswilm : sorry for the delay, merged. Thanks.

**Comment by madEng84** [comment]
@izimobil I cannot find this feature in pypy version (`==0.8.1`)
Can you check if last version has been released?

**Pull request merged**